### PR TITLE
Calculator UI

### PR DESCRIPTION
**Pull request by felixDev22**
**In this repo**

- [x] Deleted all the boilerplate from CRA (text, images, styles).
- [x] Created a directory called components.
- [x] Added Calculator.js file. & Calculator.css Inside components folder
- [x] Used class based components
- [x]  Imported Calculator App.js main component
- [x] Tested it in the browser

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,29 +1,25 @@
-import logo from './logo.svg';
+/* eslint-disable jsx-quotes */
+/* eslint-disable import/extensions */
+
+import React from 'react';
 import './App.css';
+import Calculator from './components/Calculator';
+
```
Comment (AmaduKamara) [inline src/App.js]: - [x] Let's try not to disable Linter checks as they help us write better and more professional code. Disabling Linters defies its purpose. Please fix the error 👍🏽
- [x] The import for the Calculator component from the components directory is in uppercase but the file is actually in lowercase and for that reason, the wrong import is being done and the app is not starting or loading that page. Kindly work on that 👍🏽

File: src/components/calculator.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
+/* eslint-disable react/self-closing-comp */
+/* eslint-disable react/button-has-type */
+/* eslint-disable jsx-quotes */
+import React from 'react';
+
+import './Calculator.css';
+
```
Comment (AmaduKamara) [inline src/components/calculator.js]: - [x] Let's try not to disable Linter checks as they help us write better and more professional code. Disabling Linters defies its purpose. Please fix the error 👍🏽
- [x] The import for the calculator CSS file from the components directory is in uppercase but the file is actually in lowercase and for that reason, the wrong import is being done and the app is not starting or loading the css. Kindly work on that 👍🏽

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️ ♻️ ♻️

Hi @felixDev22,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✅ You used Gitflow correctly 👍🏽
✅ Linter checks passed 👍🏽
✅ Used class-based components for Calculator and App 👍🏽
✅ Good commit messages 👍🏽
✅ Good descriptive PR 👍🏽

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/components/calculator.js (hunk 2 above)
Comment (felixDev22) [reply src/components/calculator.js]: Thanks for your suggestions. 

I have a problem every time I push the code, but it is working on my computer and other repositories. I checked for errors on my vs - npx eslint .  and no mistake shows.

I have also run npm start - and the server is running without any error. (see screenshot below)

Please advise or help me with the way forward.

ESlint error on pr - Screenshot 2023-01-11 at 2 33 27 PM


Screenshot 2023-01-11 at 2 28 42 PM

Screenshot 2023-01-11 at 2 30 21 PM

Screenshot 2023-01-11 at 2 30 53 PM

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @felixDev22,

✔️ Use of class components

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**